core/assets/bundles/bundle_nl.properties

```diff
@@ -118,619 +134,654 @@ players = {0} Spelers online
+mode.custom = Aangepaste regels
+rules.infiniteresources = Oneindige Resources
+rules.reactorexplosions = Ontploffende Reactors
+rules.wavetimer = Ronde timer
+rules.waves = Rondes
+rules.attack = Aanval modus
+rules.enemyCheat = Oneindige AI grondstoffen
 rules.unitdrops = Unit Drops
-rules.unitbuildspeedmultiplier = Unit Creation Speed Multiplier
-rules.unithealthmultiplier = Unit Health Multiplier
-rules.playerhealthmultiplier = Player Health Multiplier
-rules.playerdamagemultiplier = Player Damage Multiplier
-rules.unitdamagemultiplier = Unit Damage Multiplier
-rules.enemycorebuildradius = Enemy Core No-Build Radius:[LIGHT_GRAY] (tiles)
-rules.respawntime = Respawn Time:[LIGHT_GRAY] (sec)
-rules.wavespacing = Wave Spacing:[LIGHT_GRAY] (sec)
-rules.buildcostmultiplier = Build Cost Multiplier
-rules.buildspeedmultiplier = Build Speed Multiplier
-rules.waitForWaveToEnd = Waves wait for enemies
-rules.dropzoneradius = Drop Zone Radius:[LIGHT_GRAY] (tiles)
-rules.respawns = Max respawns per wave
-rules.limitedRespawns = Limit Respawns
-rules.title.waves = Waves
-rules.title.respawns = Respawns
-rules.title.resourcesbuilding = Resources & Building
-rules.title.player = Players
-rules.title.enemy = Enemies
+rules.unitbuildspeedmultiplier = Unit Spawn Snelheid Vermenigvulder
+rules.unithealthmultiplier = Unit Health Vermenigvulder
+rules.blockhealthmultiplier = Blok Health Vermenigvulder
+rules.playerhealthmultiplier = Speler Health Vermenigvulder
+rules.playerdamagemultiplier = Speler Damage Vermenigvulder
+rules.unitdamagemultiplier = Unit Damage Vermenigvulder
+rules.enemycorebuildradius = Niet-Bouw Afstand Vijandelijke Cores:[LIGHT_GRAY] (tegels)
+rules.respawntime = Herspawn Tijd:[LIGHT_GRAY] (sec)
+rules.wavespacing = Tijd Tussen Rondes:[LIGHT_GRAY] (sec)
+rules.buildcostmultiplier = Bouw kosten Vermenigvulder
+rules.buildspeedmultiplier = Bouw snelheid Vermenigvulder
+rules.waitForWaveToEnd = Rondes wachten tot alles is verslagen
+rules.dropzoneradius = Vijandelijke Spawn Diameter:[LIGHT_GRAY] (tegels)
+rules.respawns = Maximale Levens Per Ronde
+rules.limitedRespawns = Maximale Levens
+rules.title.waves = Rondes
+rules.title.respawns = Respawn
+rules.title.resourcesbuilding = Grondstoffen & Bouwen
+rules.title.player = Spelers
+rules.title.enemy = Tegenstanders
 rules.title.unit = Units
+rules.title.experimental = Experimenteel
+rules.lighting = Belichting
+rules.ambientlight = Mist
 content.item.name = Items
-content.liquid.name = Liquids
+content.liquid.name = Vloeisof
 content.unit.name = Units
-content.block.name = Blocks
-content.mech.name = Mechs
-item.copper.name = Copper
-item.lead.name = Lead
-item.coal.name = Coal
-item.graphite.name = Graphite
+content.block.name = Blokken
+content.mech.name = Behuizing
+item.copper.name = Coper
+item.lead.name = Lood
+item.coal.name = Steenkool
+item.graphite.name = Grafiet
 item.titanium.name = Titanium
 item.thorium.name = Thorium
-item.silicon.name = Silicon
+item.silicon.name = Silicium
```

Comment:
Nee! Silicoon en Silicium zijn twee verschillende dingen! (Silicium is wel een bestandeel van Silicoon)

Reply:
0,o